core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -184,6 +191,30 @@ static long validateDefaultMaxRequestLength(long defaultMaxRequestLength) {
         return defaultMaxRequestLength;
     }
 
+    static int validateDefaultMaxInitialLineLength(int defaultMaxInitialLineLength) {
+        if (defaultMaxInitialLineLength < 0) {
+            throw new IllegalArgumentException(
+                    "defaultMaxInitialLineLength: " + defaultMaxInitialLineLength + " (expected: >= 0)");
+        }
+        return defaultMaxInitialLineLength;
+    }
+
+    static int validateDefaultMaxHeaderSize(int defaultMaxHeaderSize) {
+        if (defaultMaxHeaderSize < 0) {
+            throw new IllegalArgumentException(
+                    "defaultMaxHeaderSize: " + defaultMaxHeaderSize + " (expected: >= 0)");
+        }
+        return defaultMaxHeaderSize;
+    }
+
+    static int validateDefaultMaxChunkSize(int defaultMaxChunkSize) {
```

Comment:
These 3 validation methods could be extracted into `validateNonNegative(int value, String fieldName)`?